packages/next/src/server/app-render/create-component-tree.tsx

```diff
@@ -195,6 +203,26 @@ async function createComponentTreeInternal({
       })
     : []
 
+  const [Forbidden, forbiddenStyles] = forbidden
+    ? await createComponentStylesAndScripts({
+        ctx,
+        filePath: forbidden[1],
+        getComponent: forbidden[0],
+        injectedCSS: injectedCSSWithCurrentLayout,
+        injectedJS: injectedJSWithCurrentLayout,
+      })
+    : []
```

Comment:
why does this not just create the component to render? it seems this needs to be turned into a ReactNode later on but this really can just be abstracted away inside this implementation